wagtail/wagtailimages/tests/tests.py

```diff
@@ -354,6 +361,36 @@ def test_get_missing_source_image_file(self):
         self.assertEqual(response.status_code, 410)
 
 
+class TestFrontendSendfileView(TestCase):
+
+    def setUp(self):
+        self.image = Image.objects.create(
+            title="Test image",
+            file=get_test_image_file(),
+        )
+
+    @override_settings(SENDFILE_BACKEND='sendfile.backends.development')
+    @unittest.skipIf(not sendfile_mod, 'Missing django-sendfile app.')
+    def test_sendfile_nobackend(self):
+        signature = generate_signature(self.image.id, 'fill-800x600')
+        response = self.client.get(reverse('wagtailimages_sendfile',
+                                           args=(signature, self.image.id,
+                                                 'fill-800x600')))
+
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response['Content-Type'], 'image/png')
+
+    @override_settings(SENDFILE_BACKEND='sendfile.backends.development')
```

Comment:
Does this setting get read anywhere? (I think we would want to use whatever the user has set in this setting as the default backend)